src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -5,13 +5,13 @@
 import mate.jdbc.model.Driver;
 
 public interface DriverDao {
-    Driver create(Driver driver);
+    public Driver create(Driver driver);
```

Comment:
we don't need use access modifers in interface

Reply:
ok
agree